homeassistant/components/light/plum_lightpad.py

```diff
@@ -0,0 +1,251 @@
+        self._load = load
+        self._brightness = load.level
+
+        self._load.add_event_listener('dimmerchange', self.dimmerchange)
+
+    def dimmerchange(self, event):
+        """Change event handler updating the brightness."""
+        self._brightness = event['level']
+        self.schedule_update_ha_state()
+
+    @property
+    def llid(self):
+        """Return the Logical Load ID (llid) associated with the load."""
+        return self._load.llid
+
+    @property
+    def should_poll(self):
+        """No polling needed."""
+        return False
+
+    @property
+    def name(self):
+        """Return the name of the switch if any."""
+        return self._load.name
+
+    @property
+    def brightness(self) -> int:
+        """Return the brightness of this switch between 0..255."""
+        return self._brightness
+
+    @property
+    def is_on(self) -> bool:
+        """Return true if light is on."""
+        return self._brightness > 0
+
+    @property
+    def dimmable(self):
+        """Return whether the load is configured as dimmable."""
+        return self._load.dimmable
+
+    @property
+    def device_state_attributes(self):
+        """Additional State attributes."""
+        return {
+            'llid': self.llid,
+            'brightness': self.brightness,
+            'dimmable': self.dimmable
+        }
+
+    @property
+    def supported_features(self):
+        """Flag supported features."""
+        if self.dimmable:
+            return SUPPORT_BRIGHTNESS
+        return None
+
+    def turn_on(self, **kwargs):
+        """Turn the light on."""
+        if ATTR_BRIGHTNESS in kwargs:
+            self._brightness = kwargs[ATTR_BRIGHTNESS]
+            self._load.turn_on(self._brightness)
+        else:
+            self._load.turn_on()
+
+    def turn_off(self, **kwargs):
+        """Turn the light off."""
+        self._load.turn_off()
+
+
+class GlowRing(Light):
+    """Represenation of a Plum Lightpad dimmer glow ring."""
+
+    def __init__(self, lightpad):
+        """Initialize the light."""
+        self._lightpad = lightpad
+        self._name = lightpad.friendly_name + " Glow Ring"
+        self._red = lightpad.glow_color['red']
+        self._green = lightpad.glow_color['green']
+        self._blue = lightpad.glow_color['blue']
+        self._white = lightpad.glow_color['white']
+        self._brightness = lightpad.glow_intensity * 255.0
+        lightpad.add_event_listener('configchange', self.configchange_event)
+
+    def configchange_event(self, event):
+        """Configuration change event handling."""
+        config = event['changes']
+        self._red = config['glowColor']['red']
+        self._green = config['glowColor']['green']
+        self._blue = config['glowColor']['blue']
+        self._white = config['glowColor']['white']
+        self._brightness = config['glowIntensity'] * 255.0
+        self.schedule_update_ha_state()
+
+    @property
+    def red(self):
```

Comment:
Red, green and blue properties don't seem necessary.

Reply:
It was helpful for debug purposes, but you are probably right that it might be overkill in the final polish